etc/input.conf

```
@@ -119,9 +121,8 @@
 #Alt+1 set current-window-scale 1.0     # reset the window size
 #Alt+2 set current-window-scale 2.0     # double the window size
 #d cycle deinterlace                    # toggle the deinterlacing filter
-#r add sub-pos -1                       # move subtitles up
-#R add sub-pos +1                       # move subtitles down
-#t add sub-pos +1                       # move subtitles down
+#r cycle-values video-rotate 270 180 90 0 # rotate left
+#t cycle-values video-rotate 90 180 270 0 # rotate right
```

```suggestion
#r cycle-values video-rotate 90 180 270 0 # rotate clockwise 
#R cycle-values video-rotate 270 180 90 0 # rotate counter-clockwise
```

I think rotation clockwise is more intuitive. Also no need for another key, we can use shift.

---

So intuitive that you inverted clockwise and counter-clockwise. :P On Google search, "rotate right" image has 3x the search results of "rotate clockwise" image, but "rotate clockwise" video has more search results than "rotate right" video. Either is fine but I feel that rotate right requires less mental gymnastics to understand.

Shift+r is fine too, but t is unbound and as I wrote in the commit message I like how intuitive it is that the key to the left rotates to the left and the key to the right rotates to the right. Guess it's up to maintainers to decide which to use.

---

r/R makes it inconsistent with the rest of the layout, even if I agree that it's better. We don't do 1/! for contrast +/-. It's 1/2...

---

We do have it both ways with w and e/W for panscan and z and x/Z for sub-delay. We could also bind both R and t this way.

---

That's fair point. Just for rotate r/R is common shortcut. gwenview and eog is using ctrl+r and ctrl+shift+r. irfanview is using r and shift+r.

But if we want to keep r/t it of course make sense that r rotate to the left and t to the right. And like you said it is consistent with current status, but in the same time if you want to be consistent why not bind rotate to `ctrl+left` and `ctrl+right` to not break existing r/t binding?

Anyway I always felt that default mpv key binds are using to much keys, there is not much space for custom command without modifiers.

---

sxiv for example uses < and >, with the same logic that the left key rotates left and the right key rotates right.

ctrl+left and ctrl+right are bound to sub-seek.

I'm just gonna bind both and R t which is consistent with the sub-pos, panscan and sub-delay bindings and should please everyone. Thankfully w r and z are bound to decrements so having r decrease video-rotate/rotate left is consistent.

---

I went with rotate clockwise/counter-clockwise which is maybe more accurate in the manual which is supposed to be more detailed, and kept the simpler and intuitive rotate right in the comment for the stats page.